.drone.yml

```diff
@@ -319,25 +278,39 @@ steps:
         path: /var/lib/mysql
     commands:
       - sudo chown -R ci:ci .
-      - /entrypoint.sh docker/ci/scripts/prepare_ui_tests.sh
+      - /entrypoint.sh docker/ci/scripts/prepare_ci_tests.sh
 
-  - name: cache-staging-misc-build-artifacts
+  # Optional optimization to reduce the size of the cache file we're about to
+  # generate. We manually delete a few large directories created by the build
+  # process which are not necessary to actually run the tests; local caches,
+  # logs, node_modules, etc.
+  - name: sanitize-cache
+    image: codedotorg/code-dot-org:1.16
+    commands:
+      - rm -r ./.venv/ # 214M
+      - rm -r ./apps/.yarn/ # 358M
+      - rm -r ./dashboard/tmp/ # 494M
+      - rm -r ./frontend/node_modules/ # 1.9G (!!!)
+      - rm -r ./frontend/.yarn/ # necessary to remove node_modules
+      - rm -r ./frontend/*/*/node_modules/ # ~115M
```

Comment:
Super open to feedback on whether y'all think this optimization is worth the added complexity. The primary benefit here is speeding up the `get-cached-staging` operation; removing these directories (`frontend/node_modules` in particular) shaves ~5mins off of the average time to start running tests.